modules/mod_ginger_base/templates/admin_seo.tpl

```diff
@@ -0,0 +1,109 @@
+{% overrules %}
+
+{% block content %}
+
+    <div class="admin-header">
+        <h2>{_ Search Engine Optimization _}</h2>
+        <p>{_ Here you find settings to optimize the indexing of your site by search engines. _}</p>
+    </div>
+
+    {% wire id="admin-seo" type="submit" postback="admin_seo" %}
+    <form name="admin-seo" id="admin-seo" method="POST" action="postback">
+
+        <div class="row">
+            <div class="col-md-7">
+
+                <div class="widget">
+                    <h3 class="widget-header">{_ General SEO Optimization _}</h3>
+                    <div class="widget-content">
+                        <div class="form-group row">
+                            <label class="control-label col-md-4" for="seo-keywords">{_ Keywords _}</label>
+                            <div class="col-md-8">
+                                <input type="text" id="seo-keywords" name="seo-keywords" value="{{ m.config.seo.keywords.value|escape }}" class="form-control" />
+                                <p class="help-block">
+                                    {_ Keywords to include on every page. Separate keywords with a "," _}
+                                </p>
+                            </div>
+                        </div>
+
+                        <div class="form-group row">
+                            <label class="control-label col-md-4" for="seo-description">{_ Description _}</label>
+                            <div class="col-md-8">
+                                <input type="text" id="seo-description" name="seo-description" value="{{ m.config.seo.description.value|escape }}" class="form-control" />
+                                <p class="help-block">
+                                    {_ Description to include on every page _}
+                                </p>
+                            </div>
+                        </div>
+
+                        <div class="form-group row">
+                            <div class="col-md-8 col-md-offset-4">
+                                <label class="checkbox-inline" for="seo-noindex" title="{_ Add a noindex, nofollow element to all pages. _}">
+                                    <input type="checkbox" id="seo-noindex" name="seo-noindex" value="1" {% if m.config.seo.noindex.value %}checked="checked"{% endif %} disabled />
+                                    {_ Exclude this site from search engines _}. {_ This setting can only be changed in the site config file. _}
```

Comment:
Opted to show a disabled checkbox instead of removing it altogether, so people won’t be looking around for something they can’t find. 😉

Reply:
I prefer that to removing it, too.